### PR TITLE
Improve spawning UX

### PR DESCRIPTION
**Pull request by khooroko**
**Description**

- [`94ed048`](https://github.com/VRFYP2019/Not-Dodgeball/pull/18/commits/94ed0487f5a91e83cd36cae3ea76de964fa251d1) adds a screen to the arm that is only visible when the player is looking more or less directly at it with the left arm rotated to face the screen at the camera.
bothhandsscreen

- [`f04ee3e`](https://github.com/VRFYP2019/Not-Dodgeball/pull/18/commits/f04ee3e215b741452a0a3133efe52cec32322f21) allows both hands to switch between spawn and tool at any time.
bothhandsspawn

- [`754bb27`](https://github.com/VRFYP2019/Not-Dodgeball/pull/18/commits/754bb27e1bdb48b6f68709b55cd1dda7290319b9) increases throwing force (and also fix some variable scoping).
throwfar

- [`f649001`](https://github.com/VRFYP2019/Not-Dodgeball/pull/18/commits/f6490013265e0007e36b51f8ce07b988db423fa7) fixes the spawning logic to allow spawning on both hands, and also display the correct number on the arm screens

@lyhvictoria

**Impact**
- [x] Medium

**Risks**
Touched `CameraRig.prefab`

**Test Plan**
Checkout `ImproveSpawnScene` and test these

### CODE REVIEW

File: Assets/Scripts/ArmScreen.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        } else {
+            trans.localScale = new Vector3(scaleX, scaleY, scaleZ);
+        }
+        text.text = "Balls: " + BallManager.Instance.playerBallQueues[0].childCount.ToString();
+        float a = Vector3.Dot(transform.forward, cam.transform.forward) > 0.5f
+            && !(Mathf.Abs(Vector3.Dot(transform.right, cam.transform.up)) < 0.8f) ? Vector3.Dot(transform.forward, cam.transform.forward) : 0;
+        ChangeTransparency(a);
+    }
+
+    private void ChangeTransparency(float alpha) {
```
Comment (lyhvictoria) [inline Assets/Scripts/ArmScreen.cs:38]: We shod prob move this to a commons soon
Something like `public void SetColor (Image image, Color c, float alpha)` or `public Color SetColor (Color c, float alpha)`

**Review by lyhvictoria — APPROVED** [review]

File: Assets/Scripts/ArmScreen.cs (hunk 1 above)
Comment (khooroko) [reply Assets/Scripts/ArmScreen.cs:38]: yea `public Color SetTransparency(Color c, float alpha)` sounds about right

Comment (lyhvictoria) [reply Assets/Scripts/ArmScreen.cs:38]: Its not really setting transparency if you can also change the color as well depending on what color you pass in

`public void SetTransparency (Image i, float a)` is closer to that function

Comment (khooroko) [reply Assets/Scripts/ArmScreen.cs:38]: but considering it's taking an `alpha` the method signature kinda suggests it should be used for setting transparency, maybe you want `public Color SetColor(Color c)`?

Comment (lyhvictoria) [reply Assets/Scripts/ArmScreen.cs:38]: Hmm, true.

**Pull request merged**